openpype/hosts/blender/plugins/load/load_look_blend.py

```diff
@@ -96,3 +96,55 @@ def _remove_container(container: Dict) -> bool:
         plugin.orphans_purge()
 
         return True
+
+    def exec_update(
+        self, container: Dict, representation: Dict
+    ) -> Tuple[str, Union[bpy.types.Collection, bpy.types.Object]]:
```

Comment:
Can we have the same ouput as the class `plugin.AssetLoader`
`-> Union[bpy.types.Collection, bpy.types.Object]`
and
`return asset_group`